app/middlewares/errors.js

```diff
@@ -1,14 +1,23 @@
 const errors = require('../errors'),
   logger = require('../logger');
 
+const statusCodes = {
+  [errors.INVALID_USER]: 400,
+  [errors.BOOK_NOT_FOUND]: 404,
+  [errors.SAVING_ERROR]: 400,
+  [errors.DATABASE_ERROR]: 503,
+  [errors.DEFAULT_ERROR]: 500
+};
+
 exports.handle = (error, req, res, next) => {
-  if (error && error.statusCode) {
-    res.status(error.statusCode);
+  const statusCode = statusCodes[error.internalCode];
+  if (statusCode) {
```

Comment:
copy-pasta from the other PR

It exists the possibility of someone adding a error to the `app/errors.js` but not adding it to the `statusCodes` map of above. In that case we are going to send that error to rollbar.
Maybe we should do something like:
```
if (error.internalCode) {
  res.status(statusCodes[error.internalCode] || 500);
} else {
  // Unrecognized error, notifying it to rollbar.
  next(error);
  res.status(500);
}
```

Reply:
Sure @mpizzagalli